app/javascript/survey/components/Feedback.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <label class="text-base font-medium text-black-800">
+      {{ $t('SURVEY.FEEDBACK.LABEL') }}
+    </label>
+    <text-area
+      v-model="feedback"
+      class="my-5"
+      :placeholder="$t('SURVEY.FEEDBACK.PLACEHOLDER')"
+    />
+    <custom-button
+      class="flex items-center font-medium float-right"
```

Comment:
We could avoid float classes as we have flex styles. We could wrap it in a div to float it right.